buildbot/configure.py

```diff
@@ -63,10 +64,12 @@ def do_configure(args):
         "-DCMAKE_BUILD_TYPE={}".format(args.build_type),
         "-DLLVM_ENABLE_ASSERTIONS={}".format(llvm_enable_assertions),
         "-DLLVM_TARGETS_TO_BUILD={}".format(llvm_targets_to_build),
-        "-DLLVM_EXTERNAL_PROJECTS=sycl;llvm-spirv;opencl-aot;xpti;libdevice",
+        "-DLLVM_EXTERNAL_PROJECTS=sycl;llvm-spirv;opencl-aot;xpti;xptifw;libdevice",
         "-DLLVM_EXTERNAL_SYCL_SOURCE_DIR={}".format(sycl_dir),
         "-DLLVM_EXTERNAL_LLVM_SPIRV_SOURCE_DIR={}".format(spirv_dir),
         "-DLLVM_EXTERNAL_XPTI_SOURCE_DIR={}".format(xpti_dir),
```

Comment:
Previous builds used "-DLLVM_EXTERNAL_XPTI_SOURCE_DIR=$SYCL_HOME/xpti"  variable to enable 'xpti'  in LLVM. I see two variables now "LLVM_EXTERNAL_XPTI_SOURCE_DIR" and "XPTI_SOURCE_DIR" - is there a reason the second one was added? It might affect scripts people may have already created.

Reply:
`LLVM_EXTERNAL_XPTI_SOURCE_DIR` tells LLVM where to find `xpti` external project, so that it could load it. It is required, because LLVM is a monorepo of multiple projects, and LLVM needs to somehow know where to find non-standard sub-projects. It shouldn't affect other scripts.